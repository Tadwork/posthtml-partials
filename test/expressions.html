<mixin name="say" from items>
	<p>Hello from {{ from }}!</p>
	<each loop="item in {{ items }}">
		<p>{{item}}</p>
	</each>
</mixin>

<div>
	<mixin name="say" from="me" items="['a', 'b', 'c']"></mixin>
</div>
