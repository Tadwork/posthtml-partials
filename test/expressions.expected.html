

<div>
	
	<p>Hello from me!</p>
	
		<p>a</p>
	
		<p>b</p>
	
		<p>c</p>
	

</div>
